### DISCUSSION TITLE
metadata获取所有表，备注获取不到

### DISCUSSION THREAD

**Discussion by dofindme** [apache/doris#15270] [A - General / Q&A]
<img width="989" alt="image" src="https://user-images.githubusercontent.com/18459525/209051938-935d411c-6a05-4b41-801f-5be8c9a6117c.png">
使用metadata查询db里的所有表，能查询出表，但是查不出来表的备注！
<img width="383" alt="image" src="https://user-images.githubusercontent.com/18459525/209052100-fbb83dc5-d65d-4a68-a5b3-d1cb7d4ef5b5.png">
但是 客户端能正常看到
<img width="610" alt="image" src="https://user-images.githubusercontent.com/18459525/209052185-bb6234da-c65c-4ca4-b0ce-b45fea854ead.png">
我尝试在jdbcurl加上参数useInformationSchema=true后，就查不到表了，想问下为什么出现这样的问题，还是使用方法有问题。

初始化代码

`DruidDataSource dataSource = new DruidDataSource();
            dataSource.setUrl("jdbc:mysql://xxxx:9030/xxxx?useInformationSchema=false");
        dataSource.setUsername("xxx");
        dataSource.setPassword("xxxxxxx");
        dataSource.setValidationQuery("SELECT 1");//用来检测连接是否有效
        dataSource.setTestOnBorrow(false);
        dataSource.setTestOnReturn(false);
        dataSource.setTestWhileIdle(true);
        dataSource.setTimeBetweenEvictionRunsMillis(60000);
        dataSource.setMaxActive(20);
        dataSource.setKillWhenSocketReadTimeout(true);
        dataSource.setUseGlobalDataSourceStat(true);
        dataSource.setAccessToUnderlyingConnectionAllowed(true);
        dataSource.setInitialSize(5);
        return dataSource;`

**Comment by JNSimba** [comment]
you can try this:  `select TABLE_NAME ,TABLE_COMMENT from information_schema.tables`
